pkg/cli/cli.go

```diff
@@ -51,9 +51,10 @@ A very simple cli.`,
 
 	rootCmd.AddCommand(
 		version(kubeClient),
-		pods(kubeClient),        // dev only, TODO remove
-		projects(dynamicClient), // dev only, TODO remove
+		pods(kubeClient),
+		get(client),
 		create(client),
+		delete(client),
```

Comment:
If the functions are creating commands it's better to explicit this with the `NewXXXCmd` (or `newXXXCmd` if you want to keep them unexported.)